crates/pathfinder/src/storage/state.rs

```diff
@@ -2172,8 +2172,8 @@ mod tests {
 
             const BLOCK_NUMBER: usize = 2;
             let filter = StarknetEventFilter {
-                from_block: Some(StarknetBlockNumber(BLOCK_NUMBER as u64)),
-                to_block: Some(StarknetBlockNumber(BLOCK_NUMBER as u64)),
+                from_block: Some(StarknetBlockNumber::new(BLOCK_NUMBER as u64).unwrap()),
+                to_block: Some(StarknetBlockNumber::new(BLOCK_NUMBER as u64).unwrap()),
```

Comment:
I think probably a grep+replace fix?

Should probably use `new_or_panic` instead, or just `new` to cheat on the option wrapping :) I'd prefer the former.

Similar in other changes in this file.

Reply:
These are probably manual early ones before I settled on `new_or_panic`, can try to find this again.

Reply:
Done in [f6c3aaa](https://github.com/eqlabs/pathfinder/pull/510/commits/f6c3aaa1729775743d6a368c478f5186f17b0f65)